dashboard/cron.py

```diff
@@ -226,24 +230,25 @@ def update_with_bq_access(self):
                     for table in tables:
                         if ("enriched_events" == table.table_id):
                             logger.debug('\t{}'.format("found table"))
-                            # loop through multiple course ids
-                            for UDW_course_id in Course.objects.get_supported_courses():
+                            # loop through multiple course ids, 10 at a time
```

Comment:
The comment mentioned "10 at a time", but I did not see the limit specified elsewhere.

Reply:
This actually increased to 20 by default, it's in the split_list method. I can be more explicit about this comment.